components/Branding/Logo.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Typography } from "@mui/material";
+
+export const Logo = ({ sx }) => {
+  return (
+    <Typography variant="h1" sx={{ ...sx }}>
```

Comment:
no need to destructor sx

Reply:
It's used to pass the style directly to original element.